src/VirtoCommerce.CustomerModule.Core/ModuleConstants.cs

```diff
@@ -50,13 +50,25 @@ public static class General
                     DefaultValue = default(DateTime),
                 };
 
+                public static SettingDescriptor EventBasedIndexation = new SettingDescriptor
```

Comment:
When you migrate settings you forgot to migrate they localization.  
See this file https://github.com/VirtoCommerce/vc-module-customer/blob/release/3.0.0/src/VirtoCommerce.CustomerModule.Web/Localizations/en.VirtoCommerce.Customer.json#L206